app/assets/stylesheets/index.scss

```diff
@@ -64,7 +63,8 @@ h2 {
 .label {
   display: block;
   text-align: center;
-  padding: 3rem;
+  padding: .5rem;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.